MobileFilter/sections/specific_web.txt

```diff
@@ -4257,6 +4257,13 @@ popularmechanics.com#?#section > div[class*=" "]:has(> div[class]:first-child +
 veranda.com,wetter.com#?#div[class^="sc-"]:has(> div.ad-container)
 wetter.com#?#.som-wrapper:has(> div[data-slotname])
 wetter.com#?#.clickperformance-wrapper:has(> .ad-flex)
+wetter.com#?#.mbanner-fixed:remove()
+wetter.com#$#.mbanner-is-fixed { top: 0px !important; padding-top: 0px !important; }
+wetter.com#$##main-header-fixable { top: 0px !important; }
```

Comment:
I don't see `.mbanner-is-fixed` Could you please show a screenshot? Is it an ad or mobile app banner?
```
wetter.com#$#.mbanner-is-fixed { top: 0px !important; padding-top: 0px !important; }
wetter.com#$##main-header-fixable { top: 0px !important; }
```

Reply:
It's there on at least this specific page: https://www.wetter.com/deutschland/berlin/DE0001020.html

Without the rule there some empty space between the top bar and the search field:
![image](https://github.com/AdguardTeam/AdguardFilters/assets/57908117/65a12046-4f0b-4125-a3ab-82e19f95881b)

Reply:
Ok then. But they are about ads or mobile app banner?

Reply:
No, those were ads :)